src/App.js

```diff
@@ -418,7 +419,7 @@ class Loader extends React.Component<Props, State> {
       isReportMode: false,
     });
     if (this.isEditMode()) {
-      this.props.history.push(`/nodes/${this.state.featureId}`);
+      this.props.history.push(`/nodes/${String(this.state.featureId)}`);
```

Comment:
Is this required? I though template literals would do string conversion when doing string interpolation?

Reply:
Flow complains without this – strictly, `featureId` could be `undefined` which formally is not castable to String.